### PR TITLE
feat: exposed plan endpoints on ruby client

### PR DESCRIPTION
**Pull request by madhuravius**
This does in fact work, as outlined by these screenshots:

image

image

---

The plans endpoint is merged, however the active plan endpoints have not yet been merged. The active plan endpoints being treated as a top-level citizen alters the topology of the Home HAL document and really should be vetted/approved by other engineers.

### CODE REVIEW

**Review by almathew — APPROVED** [review]

**Review by almathew — COMMENTED** [review]
actually, this might not work

**Review by almathew — CHANGES_REQUESTED** [review]
doing required comment to un-review

File: lib/aptible/api/plan.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      field :id
+      field :name
+      field :container_memory_limit_mb
+      field :cpu_allowed_profiles
+      field :disk_limit_gb
+      field :ephemeral_session_limit
+      field :environment_limit
+      field :vhost_limit
+      field :compliance_dashboard_access
+      field :allowed_plans
```
Comment (madhuravius) [inline lib/aptible/api/plan.rb:13]: drop

File: lib/aptible/api/active_plan.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      field :id
+      field :plan_id
+      field :container_memory_limit_mb
+      field :cpu_allowed_profiles
+      field :disk_limit_gb
+      field :ephemeral_session_limit
+      field :environment_limit
+      field :vhost_limit
+      field :compliance_dashboard_access
+      field :available_plans
```
Comment (madhuravius) [inline lib/aptible/api/active_plan.rb:15]: correct

Comment (madhuravius) [reply lib/aptible/api/active_plan.rb:15]: image

already correct

**Review by almathew — APPROVED** [review]
nvm is fine

**Pull request merged**